pkg/kube/controllers/quarksstatefulset/quarksstatefulset_reconciler.go

```diff
@@ -108,28 +110,18 @@ func (r *ReconcileQuarksStatefulSet) Reconcile(request reconcile.Request) (recon
 	}
 
 	// Get the current StatefulSet.
-	currentStatefulSet, currentVersion, err := GetMaxStatefulSetVersion(ctx, r.client, qStatefulSet)
+	_, currentVersion, err := GetMaxStatefulSetVersion(ctx, r.client, qStatefulSet)
```

Comment:
can we delete this?

Reply:
We still need the version for the annotation. I have moved it to `calculateDesiredStatefulSets` to be less confusing.